### ISSUE TITLE
Extra elipsis when displaying a single-event EventSet

### ISSUE THREAD

**Issue by javiber** [google/temporian#410]
In this case the ellipsis (`...`) should not be displayed

<img width="222" alt="Screenshot 2024-03-26 at 2 55 10 PM" src="https://github.com/google/temporian/assets/3588715/0abae9b4-f4dc-4902-baa6-07912b06a36f">

**Comment by akshatvishu** [comment]
I think this is happening due to :

```py

            # Create ellipsis row between first half and last half if more than max_timestamps entries
            table.appendChild(html_table_row(dom, row))
            if (
                timestamp_idx == ((max_timestamps // 2) - 1)
                and num_timestamps > max_timestamps
            ) or (max_timestamps == 1):                                     # THIS IS CREATING THE ELLIPSE
                ellipsis_row = [ELLIPSIS] * (
                    1 + len(visible_feats) + int(has_hidden_feats)
                )
                table.appendChild(html_table_row(dom, ellipsis_row))
```
at here: [def display_html](<https://github.com/google/temporian/blob/main/temporian/implementation/numpy/data/display_utils.py#L192>).

Maybe we can do `max_timestamps == 1 and num_timestamps > 1`  instead of just `or (max_timestamps == 1):`

**Comment by javiber** [comment]
Yes, the `max_timestamps` is the `min(config.display_max_events, num_timestamps)` so when we fixed the ellipsis for `display_max_events == 1` we broke it when `num_timestamps == 1`
Adding that check would fix it, although I think we can factor-out the `num_timestamps > max_timestamps` and do something like:

``` python
(num_timestamps > max_timestamps) and ( (timestamp_idx == ((max_timestamps // 2) - 1)) or (max_timestamps == 1) )
``` 
and I feel that the last 2 conditions linked by the `or` could be simplified into a single one 🤔

**Comment by akshatvishu** [comment]
> Yes, the `max_timestamps` is the `min(config.display_max_events, num_timestamps)` so when we fixed the ellipsis for `display_max_events == 1` we broke it when `num_timestamps == 1` Adding that check would fix it, although I think we can factor-out the `num_timestamps > max_timestamps` and do something like:
> 
> ```python
> (num_timestamps > max_timestamps) and ( (timestamp_idx == ((max_timestamps // 2) - 1)) or (max_timestamps == 1) )
> ```
> 
> and I feel that the last 2 conditions linked by the `or` could be simplified into a single one 🤔

Can you tell me how to run `jupyter notebooks` in our venv so, that I can test it because  if I just download the `jupyter` extension from `vscode` and try to run something; the kernel just don't detect the `poetry venv`
